src/js/favorites.js

```diff
@@ -78,12 +79,14 @@ function resizerHandler(data = localData) {
 }
 
 document.addEventListener('DOMContentLoaded', () => {
-  localData = getFavoritesData();
-  // Initial run
-  resizerHandler(localData);
-  const throttledHandleResize = throttle(() => resizerHandler(localData), 50);
-  // Throttle the resizerHandler function
-  window.addEventListener('resize', throttledHandleResize);
-  favoritesListRef.addEventListener('click', onExerciseStartClick);
-  favoritesListRef.addEventListener('click', onExerciseRemoveClick);
+  if (window.location.pathname.includes('favorites.html')) {
```

Comment:
As favourites.js should be added only to favourites.html page we do not need that check. Please check if the favourites.js added only to dedicated page

Reply:
The thing is that I'm using the render method of the favourites.js in exercise popup. I know it's kinda ugly, but I haven't found a better way. And the thing is that it fails even without popup opening. Do you have a better solution?